# EstimatorTest.msg
time stamp

float64[12] Parameter

float64[12] FeetBasedPosition
float64[12] FeetBasedVelocity

float64[9] EstimatedXYZ
float64[9] EstimatedRPY

float64[12] DataCheckA
float64[12] DataCheckB
float64[12] DataCheckC
float64[12] DataCheckD
float64[12] DataCheckE